src/pages/students/Students.tsx

```diff
@@ -1,11 +1,192 @@
-import './Students.css'
+import "./Students.css";
+import { useForm } from "react-hook-form";
+import type { Resolver } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { studentsScheme } from "../students/students.scheme";
+import { MOCKED_PAYMENT_TYPES } from "../../mocks/paymentTypes";
+import { useNavigate } from "react-router-dom";
+import ErrorIcon from "../../assets/error-icon.png";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import { Controller } from "react-hook-form";
 
+type FormData = {
+  name: string;
+  lastname: string;
+  nroTelefono: string;
+  fechaNacimiento: Date | null;
+  tipoPago: string;
+  cantidadDias: number;
+  diaPago: string;
+  diasExtra: string;
+  fechaIngreso: Date | null;
+};
 function Students() {
+  const navigate = useNavigate();
+  const resolver = yupResolver(studentsScheme) as Resolver<FormData>;
+
+  const onSubmit = (data: any) => {
+    console.log("Formulario enviado con datos:", data);
+  };
+  const {
+    register,
+    handleSubmit,
+    watch,
+    control,
+    formState: { errors },
+  } = useForm<FormData>({
+    resolver,
```

Comment:
Siguiendo el comentario anterior, aca podrias hacer

```suggestion
    resolver: yupResolver(studentsScheme),
```